Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -94,7 +94,7 @@ void SettingsWindow::AddCategoryToList(const QString& title, const std::string&
 void SettingsWindow::MakeCategoryList()
 {
   m_categories = new QListWidget;
-  m_categories->setMaximumWidth(175);
+  m_categories->setFixedWidth(125);
```

Comment:
If I remove the sizeHint for QListWidgetItem, it adds a bunch of space around the icon. Not something I really want, considering that this PR is supposed to remove spacing.